pkg/k8s/resource/resource_test.go

```diff
@@ -40,7 +41,7 @@ func TestMain(m *testing.M) {
 	goleak.VerifyTestMain(m, goleak.Cleanup(cleanup))
 }
 
-func testStore(t *testing.T, node *corev1.Node, store resource.Store[*corev1.Node]) {
+func testStore(t testing.TB, node *corev1.Node, store resource.Store[*corev1.Node]) {
```

Comment:
Nit: the change is probably not required anymore? Conventionally this should be `tb` now which would lead to a lot of churn.